api/filter-alerts.go

```diff
@@ -0,0 +1,228 @@
+package api
+
+import (
+	"fmt"
+	graphql "github.com/cli/shurcooL-graphql"
+	"github.com/humio/cli/api/internal/humiographql"
+)
+
+type FilterAlert struct {
```

Comment:
Filter alerts throttling has just hit master, so we should consider whether we want to include that now, or whether we want to get a version of the API and CLI out that supports the current filter alerts API, and then later push another update.